src/main/java/paymelah/logic/parser/CliSyntax.java

```diff
@@ -11,5 +11,6 @@ public class CliSyntax {
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
+    public static final Prefix PREFIX_DEBT = new Prefix("d/");
```

Comment:
This prefix clashes with the prefix for Description (which has already been merged) 🙀

Reply:
Changed it to /debt as a placeholder for now.

Reply:
I don't think this is a huge issue, since debt-indexes and description+money are never going to be in the same context (unless we want to allow deleting debts by description in future?)

Reply:
@TheSoggy @lyuting47 what do yall think
I personally feel it's better to avoid repetition (for extensibility) but at the same time it's weird to have debt be the only prefix that needs more letters hmmm

Reply:
Hm how about no prefix? Since they are both indices. So like `deletedebt 1 /2`. Or would that be too confusing

Reply:
Not a fan of that, debt/ is fine
Or we can use a different letter like i/ for debt Index

Reply:
I'm still concerned that users may confuse the position of the index types.
What about `deletedebt DEBT_INDEX_1 DEBT_INDEX_2 of/ PERSON_INDEX` ?